src/renderer/screens/manager/AppsList/InstallSuccessBanner.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+import React, { useState, useEffect, useCallback, useRef, memo } from "react";
+
+import styled from "styled-components";
+
+import { Trans } from "react-i18next";
+
+import { getCryptoCurrencyById } from "@ledgerhq/live-common/lib/currencies";
+import { isCurrencySupported } from "@ledgerhq/live-common/lib/data/cryptocurrencies";
+
+import type { State, Action } from "@ledgerhq/live-common/lib/apps/types";
+
+import Text from "~/renderer/components/Text";
+import Box from "~/renderer/components/Box/Box";
+import FadeInOutBox from "~/renderer/components/FadeInOutBox";
+import IconCross from "~/renderer/icons/Cross";
+
+import Button from "~/renderer/components/Button";
+
+const IconContainer = styled(Box).attrs(() => ({
+  horizontal: true,
+  alignItems: "center",
+  p: 4,
+}))`
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  cursor: pointer;
+`;
+
+const LogoContainer = styled(Box).attrs(() => ({
+  flex: 1,
+  justifyContent: "center",
+  alignItems: "center",
+}))`
+  position: absolute;
+  top: 10px;
+  left: 20px;
+
+  ${IconContainer} {
+    width: 100%;
+    max-width: 110px;
+  }
+`;
+
+const Container = styled.div`
+  position: relative;
+`;
+
+type Props = {
+  state: State,
+  dispatch: Action => void,
+  isIncomplete: boolean,
+  addAccount: (*) => void,
+};
+
+let installs = new Set([]);
+
+const InstallSuccessBanner = ({ state, isIncomplete, dispatch, addAccount }: Props) => {
+  const cardRef = useRef();
+  const [installSuccess, setInstallSuccess] = useState([]);
+  const { installQueue, uninstallQueue, installed, appByName } = state;
+
+  const onAddAccount = useCallback(() => {
+    if (installSuccess[0].currencyId)
+      addAccount(getCryptoCurrencyById(installSuccess[0].currencyId));
+  }, [addAccount, installSuccess]);
+
+  const onInstallSuccess = useCallback(() => {
+    const installArray = Array.from(installs)
+      .map(n => appByName[n])
+      .filter(
+        app =>
+          app &&
+          app.currencyId &&
+          isCurrencySupported(getCryptoCurrencyById(app.currencyId)) &&
+          installed.findIndex(ins => ins.name === app.name) >= 0,
```

Comment:
the logic here is very complex. let's discuss on monday what we essentially need. from what I see, we just want that the `state` that comes from live-common keep track of **the last app that was installed and that is an opportunity to be added on live**, and that can easily be tracked in the state itself instead of keeping a state like this.

essentially we could trigger the modal when `state.opportunityToAddAccountsCurrency && state.queues are empty` (pseudo code :yolo: )